nucypher/blockchain/eth/policies.py

```diff
@@ -38,23 +41,23 @@ class BlockchainArrangement(Arrangement):
     federated = False
 
     def __init__(self,
-                 author: PolicyAuthor,
-                 miner: Miner,
+                 alice: PolicyAuthor,
+                 ursula: Miner,
```

Comment:
Is that where we're going with this?  We presume that the "Miner" ("Worker?") is always "ursula"?  I thought that, previously, we were taking care to keep these layers distinct from one another.

Reply:
Hmm, unsure how to keep pyCharm happy - I keep going back and forth... What do you think is better?

Reply:
Reading the module as a whole, I think this is fine at least for now.

Reply:
:heavy_check_mark: